### PR TITLE
add bip39 converter for haven and monero to crypto package

### PR DESCRIPTION
**Pull request by MartyHav**

### CODE REVIEW

File: packages/xchain-crypto/package.json [hunk 1]
```diff
@@ -54,6 +57,7 @@
   "dependencies": {
     "bech32": "^2.0.0",
     "bip39": "^3.0.4",
+    "bip39-converter": "^1.0.3",
```
Comment (veado) [inline packages/xchain-crypto/package.json:60]: @MartyHav Where is the source code of `bip39-converter` published? Could not find any information at https://www.npmjs.com/package/bip39-converter nor at https://github.com/Coinomi/bip39-coinomi/

Comment (veado) [reply packages/xchain-crypto/package.json:60]: Just found it https://github.com/haven-protocol-org/bip39mnemonicConverter

Pls add this repo to https://www.npmjs.com/package/bip39-converter

**Review by veado — CHANGES_REQUESTED** [review]
Thanks for your PR! After re-thinking my [previous comment](https://github.com/xchainjs/xchainjs-lib/pull/588#issuecomment-1138752823) I would suggest the following: Since `convertToHavenMnemonic` is just wrapper of `convertBip39ToHavenMnemonic` and used by `xchain-haven` only and not any other `xchain-*` package, it should be moved into `utils` of `xchain-haven`. Monero can do the same by placing it into its `util` of `xchain-monero`.

File: packages/xchain-crypto/package.json (hunk 1 above)
Comment (MartyHav) [reply packages/xchain-crypto/package.json:60]: will be added with next version

**Comment by MartyHav** [comment]
Alright, closing this PR.

**Pull request closed without merging**